[2025-01-10T16:56:05.055+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-10T16:56:05.072+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T16:55:00+00:00 [queued]>
[2025-01-10T16:56:05.081+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T16:55:00+00:00 [queued]>
[2025-01-10T16:56:05.082+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-10T16:56:05.097+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): append_to_database> on 2025-01-10 16:55:00+00:00
[2025-01-10T16:56:05.103+0000] {standard_task_runner.py:72} INFO - Started process 682 to run task
[2025-01-10T16:56:05.105+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'stock_information', 'append_to_database', 'scheduled__2025-01-10T16:55:00+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/vnstock_stream.py', '--cfg-path', '/tmp/tmpxz7_y6tv']
[2025-01-10T16:56:05.107+0000] {standard_task_runner.py:105} INFO - Job 105: Subtask append_to_database
[2025-01-10T16:56:05.146+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_information.append_to_database scheduled__2025-01-10T16:55:00+00:00 [running]> on host 453ac4defedc
[2025-01-10T16:56:05.234+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='TuanDz' AIRFLOW_CTX_DAG_ID='stock_information' AIRFLOW_CTX_TASK_ID='append_to_database' AIRFLOW_CTX_EXECUTION_DATE='2025-01-10T16:55:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-10T16:55:00+00:00'
[2025-01-10T16:56:05.237+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-10T16:56:20.277+0000] {logging_mixin.py:190} INFO - Lỗi kết nối db: ('HYT00', '[HYT00] [Microsoft][ODBC Driver 18 for SQL Server]Login timeout expired (0) (SQLDriverConnect)')
[2025-01-10T16:56:20.281+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-10T16:56:20.296+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-10T16:56:20.298+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_information, task_id=append_to_database, run_id=scheduled__2025-01-10T16:55:00+00:00, execution_date=20250110T165500, start_date=20250110T165605, end_date=20250110T165620
[2025-01-10T16:56:20.348+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-10T16:56:20.361+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-10T16:56:20.365+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
